api/src/Command/StaffUserDeletionCommand.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Command;
+
+use App\Repository\UserRepository;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class StaffUserDeletionCommand extends Command
+{
+    public static $defaultName = 'digideps:nonexistent-sirius-staff-accounts';
+
+    public static array $digidepsStaffToExcludeFromDeletion =
+        [
+            35636 => 'gugandeep.chani@digital.justice.gov.uk',
+            34368 => 'james.warren@digital.justice.gov.uk',
+            49768 => 'nicola.mcmillan@digital.justice.gov.uk',
+            46417 => 'oludara.oginni@digital.justice.gov.uk',
+            29876 => 'jack.goodby@digital.justice.gov.uk',
+            45515 => 'mia.gordon@digital.justice.gov.uk',
+            33833 => 'caroline.hufton@digital.justice.gov.uk',
+            43009 => 'jude.rattle@digital.justice.gov.uk',
+            43411 => 'tom.gulliver@digital.justice.gov.uk',
+            43177 => 'peter.scottreid@digital.justice.gov.uk',
```

Comment:
Might wanna give Danny Collins the headsup before we delete him, also...

Reply:
OK ignore that - I've just checked with Danny and he has two accounts, and we're ok to delete the one as it stands